apps/marketing/content/blog/design-system.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Open Sourcing Documenso's Design
+description: It's day 1 of our first launch week. We are kicking it off by open sourcing Documenso's design system! Let's go..
+authorName: 'Timur Ercan'
+authorImage: '/blog/blog-author-timur.jpeg'
+authorRole: 'Co-Founder'
+date: 2023-09-25
+tags:
+  - Design
+  - Open Source
+  - Community
+---
+
+<figure>
+  <MdxNextImage
+    src="/blog/designsystem.png"
+    width="1260"
+    height="630"
+    alt="Documenso's Design System"
+  />
+
+  <figcaption className="text-center">
+   Documenso's Design System ✨
+  </figcaption>
+</figure>
+
+> TLDR; Our design system is OSS under MIT at [design.documenso.com](https://design.documenso.com)
+
+Today, we are open sourcing our design system, lovingly crafted by @Thilo. The system is public on Figma and will be available at design.documenso.com from here on out.
```

Comment:
> @Thilo

Can you link the twitter handle or anything else? This doesn't link anywhere

Reply:
Same for design.documenso.com, it doesn't link anywhere

Reply:
good points

Reply:
done

Reply:
https://documen.so/design

Reply:
Right I see what you've done here